lib/src/expandable_page_view.dart

```diff
@@ -207,7 +207,9 @@ class _ExpandablePageViewState extends State<ExpandablePageView> {
     _sizes = _prepareSizes();
     _pageController = widget.controller ?? PageController();
     _pageController.addListener(_updatePage);
-    _currentPage = _pageController.initialPage.clamp(0, _sizes.length - 1);
+    final sizesLength = _sizes.length;
+    final sizesUpperBounds = sizesLength == 0 ? 0 : sizesLength - 1;
```

Comment:
I'd generally advise against creating pointless variables for this. It doesn't save any space and it prevents the linter from telling you to use `isEmpty` instead of `.length == 0`